.vscodeignore

```diff
@@ -8,4 +10,5 @@ sonar-project.properties
 tsconfig.json
```

Comment:
as "**/tsconfig.json" is added, I think this line can be removed

Reply:
I have also added `.ui-testing` and `out/ui-test/**` to `.vscodeignore`.